pkg/k8shandler/collection.go

```diff
@@ -92,7 +92,7 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateCollection() (err err
 		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
 			if !reflect.DeepEqual(fluentdStatus, cluster.Status.Collection.Logs.FluentdStatus) {
 				if printUpdateMessage {
-					logrus.Info("Updating status of Fluentd")
+					logrus.Infof("Updating status of Fluentd")
```

Comment:
we aren't formatting.. is there an error from using `Info`?

Reply:
Sorry, I used it for debugging and failed to undo it. :p  Fixed.